src/main.rs

```diff
@@ -153,11 +153,13 @@ fn run() -> Result<(), failure::Error> {
 
     if let Some(_matches) = matches.subcommand_matches("config") {
         println!("Enter email: ");
-        let email: String = read!("{}\n");
+        let mut email: String = read!("{}\n");
+        email.truncate(email.trim_end().len());
```

Comment:
the truncate dance here is to avoid allocating an extra string, and to keep email as an owned String instead of a string slice